src/pluto/components/null_graph.py

```diff
@@ -0,0 +1,186 @@
+import streamlit as st
+import pandas as pd
+import plotly.graph_objects as go
+import plotly.express as px
+from src.constants import COLOR_SCHEME
+import pdb
+
+
+class NullReport:
+    def __init__(self, data, v1, v2, v3, condo, mapped):
+        self.df_null = data
+        self.v1 = v1
+        self.v2 = v2
+        self.v3 = v3
+        self.condo = condo
+        self.mapped = mapped
+
+    def __call__(self):
+        st.subheader("Null Graph")
+        field_range = st.slider(
+            "Select a range of values",
```

Comment:
I think better text would be "Select which fields to display (ranked from most to least nulls)"

Reply:
Sorry, before I didn't rank all of them. I just displayed selected fields in order to make the graph look nicer. The most recent commit ranks all fields in descending order and displays them in order as well. But the first one is quite annoying, as the number is really large, which makes the graph very unbalanced.